### PR TITLE
Multi scale

### PR DESCRIPTION
**Pull request by tink2123**
support mutli-scale training for rec model

### CODE REVIEW

File: configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml [hunk 1]
```diff
@@ -90,20 +91,26 @@ Train:
...
         keep_keys:
         - image
         - label_ctc
         - label_gtc
         - length
         - valid_ratio
+  sampler:
+    name: MultiScaleSampler
+    scales: [[320, 32], [320, 48], [320, 64]]
+    # divide_factor: to ensure the width and height dimensions can be devided by downsampling multiple
```
Comment (andyjpaddle) [inline configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml:104]: 去掉注释

File: configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml [hunk 2]
```diff
@@ -128,5 +139,5 @@ Eval:
   loader:
     shuffle: false
     drop_last: false
-    batch_size_per_card: 128
+    batch_size_per_card: 1
```
Comment (andyjpaddle) [inline configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml:142]: bs只能为1？

File: configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml (hunk 1 above)
Comment (tink2123) [reply configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml:104]: done, thx

File: configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml (hunk 2 above)
Comment (tink2123) [reply configs/rec/PP-OCRv4/ch_PP-OCRv4_rec.yml:141]: eval时变长预测，bs只能是1

**Review by andyjpaddle — APPROVED** [review]

**Pull request merged**